mock-api/snapshot.ts

```diff
@@ -77,7 +77,7 @@ export const snapshots: Json<Snapshot>[] = [
     state: 'ready',
   },
   {
-    id: '7fc6ca11-452e-d3e4-9e1c-752ff615abea',
```

Comment:
Turns out this was an invalid UUID, correctly rejected by Zod because of the `d` in `d3e4`, which is supposed to be a 4 for UUID v4.